cfgov/mega_menu/tests/test_frontend_conversion.py

```diff
@@ -13,6 +13,8 @@ class FrontendConverterTests(TestCase):
     @classmethod
     def setUpTestData(cls):
         cls.root_page = Site.objects.get(is_default_site=True).root_page
+        cls.request = RequestFactory().get('/consumer-tools/')
+        Site.find_for_request(cls.request)
```

Comment:
Same as above, I actually think we could remove this if you're not actually using the Site that's returned. That would reduce the query count described below.

Reply:
According to @chosak adding the `find_for_request` was his recommendation to reduce the query count.

Removing the `find_for_request` will increase the query count and produce the following:
AssertionError: 3 != 2 : 3 queries executed, 2 expected

Refer to https://docs.wagtail.io/en/stable/releases/2.9.html#sitemiddleware-and-request-site-deprecated
Please let me know how you want me to proceed.